wzdx/monitoring/monitor.py

```diff
@@ -0,0 +1,66 @@
+import os
+import json
+import requests
+import uuid
+import time
+from google.cloud import monitoring_v3
+
+WZDX_REST_ENDPOINT_PROD = os.getenv(
+    'WZDX_REST_ENDPOINT_PROD', 'https://data.cotrip.org/api/v1/wzdx?apiKey={api_key}')
+WZDX_REST_API_KEY_PROD = os.getenv(
+    'WZDX_REST_API_KEY_PROD', 'CT0E0KD-1S1MKYA-QSJ8WV7-045RH37')
+WZDX_REST_ENDPOINT_TEST = os.getenv(
+    'WZDX_REST_ENDPOINT_TEST', 'https://test.data.cotrip.org/api/v1/wzdx?apiKey={api_key}')
+WZDX_REST_API_KEY_TEST = os.getenv(
+    'WZDX_REST_API_KEY_TEST', 'CT0E0KD-1S1MKYA-QSJ8WV7-045RH37')
+PROJECT_ID = os.getenv('PROJECT_ID', 'cdot-rtdh-dev')
+PROD_METRIC_NAME = os.getenv(
+    'PROD_METRIC_NAME', 'custom.googleapis.com/wzdx_rest_count')
+
+
+def get_api_response(url, api_key):
+    # format URL with username, password, and file path
+    formatted_url = url.format(api_key=api_key)
+
+    content = requests.get(formatted_url).content.decode("utf-8")
+    # l.info(f"API Response content length: {len(content)}")
+
+    return json.loads(content)
+
+
+def write_metric(metric_name, value):
+    client = monitoring_v3.MetricServiceClient()
+    project_name = f"projects/{PROJECT_ID}"
+
+    series = monitoring_v3.TimeSeries()
+    series.metric.type = metric_name
+    series.resource.type = "global"
+    series.metric.labels["project"] = "cdot-rtdh-dev"
```

Comment:
Should this be injected with our PROJECT_ID value instead?

Reply:
updated